### ISSUE TITLE
Print welcome message on first startup

### ISSUE THREAD

**Issue by cblgh** [cabal-club/cabal-cli#80]
Let's treat the first startup as a special case and print some stuff
* a welcome message / ascii art 
* the commands
* link to the github org, the issues tracker

**Comment by noffle** [comment]
Great idea!

**Comment by joehand** [comment]
Does the !status thing cover this? Can we close it?

**Comment by cblgh** [comment]
the !status channel will temporarily be removed once `cabal-client` is finished, so i want to keep this up lest i forget to implement it again.

(the reason being that the abstraction for channels only deals with actual channels, and the `!status` channel will need special handling which is kind of out of scope for the remaining work. but it has been considered and will be added in such a way that the desktop clients also gets them for free :)

**Comment by cblgh** [comment]
we have this! thanks cabal-client 

https://github.com/cabal-club/cabal-cli/pull/126

**Comment by cblgh** [comment]
(still want some o dat ascii tho, someone hook me up with the old school ansi scene)
